src/libostree/ostree-sepolicy.c

```diff
@@ -300,6 +303,12 @@ initable_init (GInitable     *initable,
         goto out;
       self->rootfs_dfd = self->rootfs_dfd_owned;
     }
+  else if (!self->path)
```

Comment:
Not quite, because if we only have a rootfs dfd, we need to generate `self->path`.  I reworked the code ⬇️ though to `#if 0` out the bit we weren't using and changed the evaluation order.